hash/bbc_flop_65c102.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE softwarelist SYSTEM "softwarelist.dtd">
 
-<!-- BBC Micro - 65C102 Co-Processor Discs -->
+<!-- Acorn BBC Micro - 65C102 Co-Processor Discs -->
 
 <!-- Loading Instructions:
 
       Hold down the SHIFT key and press and release the BREAK key.
 -->
 
-<softwarelist name="bbc_flop_65c102" description="65C102 Co-Processor Discs">
+<softwarelist name="bbc_flop_65c102" description="Acorn BBC Master 65C102 Co-Processor discs">
```

Comment:
Not specifically for BBC Master.

Reply:
This and the previous two should be `Acorn 32016 Co-Processor discs`, `Acorn 6502 2nd Processor discs` and `Acorn 65C102 Co-Processor discs`?

Reply:
Yeah, I agree with you about conciseness. I think this changes will allow people to easily identify these boards as Acorn computer accessories and when listed alphabetically will show next to the other Acorn computers.